### PR TITLE
fix/icrc-2-and-icrc-1

### PR DESCRIPTION
**Pull request by Julio-SDSharp**
## Fix into ICRC-1 and ICRC-2

This pull request addresses several issues and implements enhancements, including:

- Clearing allowances cache on lock
- Adjusting table font size
- Removing unnecessary allowance request
- Fixing loading GIF on transfer validations
- Adding contact asset icon
- Removing amount validation on change
- Correctly displaying "no enough balance" message for subaccounts

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-NODE_ENV=PRODUCTION
+NODE_ENV=DEV
```
Comment (OliverSDSharp) [inline .env:1]: Why Dev?

File: frontend/utils.ts [hunk 2]
```diff
@@ -124,9 +124,9 @@ export const toHoleBigInt = (numb: string, decimal: number) => {
     for (let index = 0; index < decimal; index++) {
       addZeros = "0" + addZeros;
     }
-    return BigInt(parts[0] + addZeros);
+    return BigInt(parts[0].replace(/,/g, "") + addZeros);
```
Comment (OliverSDSharp) [inline frontend/utils.ts:127]: the , means the 1,000? or means 1000,00?

**Review by OliverSDSharp — CHANGES_REQUESTED** [review]

File: frontend/utils.ts (hunk 2 above)
Comment (Jair-MoralesN) [reply frontend/utils.ts:127]: 1,000
We work only with US notation "2,123,345.56"

File: .env (hunk 1 above)
Comment (Jair-MoralesN) [reply .env:1]: PROD is for the FULL APP with HPL

File: frontend/utils.ts (hunk 2 above)
Comment (OliverSDSharp) [reply frontend/utils.ts:127]: ok. it works with big numbers? shouldnt be replaceAll?

**Pull request merged**